### ISSUE TITLE
Some info lines have no PV

### ISSUE THREAD

**Issue by dsjoerg** [jdart1/arasan-chess#16]
Hi, here's a strange behavior that I got while running 1de797f54bebaa41e62b45b4cdd08eca9e25b9fa.  So I upgraded to the current commit, figuring that your recent fixes addressed it, but nope it's still there :)

The problem is that certain info lines, such as the one with `nodes 6413`, have an empty `pv` section, so it's not possible to know which moves it's analyzing.

```
Arasan v20.5.0-39-g797ba6b Copyright 1994-2018 by Jon Dart. All Rights Reserved.
uci
id name Arasan v20.5.0-39-g797ba6b
id author Jon Dart
option name Hash type spin default 32 min 4 max 64000
option name Ponder type check default true
option name Contempt type spin default 0 min -200 max 200
option name Use tablebases type check default false
option name SyzygyTbPath type string default syzygy
option name SyzygyUse50MoveRule type check default true
option name SyzygyProbeDepth type spin default 4 min 0 max 64
option name MultiPV type spin default 1 min 1 max 10
option name OwnBook type check default true
option name Book selectivity type spin default 50 min 0 max 100
option name Threads type spin default 1 min 1 max 64
option name UCI_LimitStrength type check default false
option name UCI_Elo type spin default 2600 min 1000 max 2600
uciok
setoption name MultiPV value 5
position fen 8/8/8/pp4R1/8/5R1K/rk6/8 b - - 10 59
go depth 8
failed to open ../bin/book.bin
info multipv 1 depth 1 score cp -266 time 0 nodes 186 hashfull 0 pv b5b4 f3f2
info multipv 2 depth 1 score cp -335 time 0 nodes 195 hashfull 0 pv a5a4 g5b5
info multipv 3 depth 1 score cp -386 time 0 nodes 203 hashfull 0 pv a2a1 g5b5
info multipv 4 depth 1 score cp -390 time 0 nodes 210 hashfull 0 pv a2a4 g5b5
info multipv 5 depth 1 score cp -418 time 0 nodes 216 hashfull 0 pv b2b1 g5b5
info multipv 1 depth 2 score cp -391 time 0 nodes 790 hashfull 0 pv a2a4 g5b5 b2c2 b5c5 c2d2
info multipv 2 depth 2 score cp -420 time 0 nodes 1000 hashfull 0 pv b2b1 g5b5 b1c1
info multipv 3 depth 2 score cp -420 time 0 nodes 1175 hashfull 0 pv b2c1 g5b5
info multipv 4 depth 2 score cp -508 time 0 nodes 1320 hashfull 0 pv b2a1 g5b5
info multipv 5 depth 2 score cp -647 time 0 nodes 1435 hashfull 0 pv b5b4 f3f2 b2b3 g5g3
info multipv 1 depth 3 score cp -538 time 0 nodes 2203 hashfull 0 pv b2c1 f3f1 c1b2 g5b5 b2a3 b5a5
info multipv 2 depth 3 score cp -538 time 140440342977984 nodes 2452 nps 0 hashfull 0 pv b2a1 f3f1 a1b2 g5b5 b2a3 b5a5
info multipv 3 depth 3 score cp -541 time 140440342977984 nodes 2635 nps 0 hashfull 0 pv b2b1 g5b5 a2b2 b5a5
info multipv 4 depth 3 score cp -643 time 140440342977984 nodes 3079 nps 0 hashfull 0 pv a5a4 f3f2 b2b3 g5g3 b3c4 f2a2
info multipv 5 depth 3 score cp -647 time 140440342977984 nodes 2855 nps 0 hashfull 0 pv b5b4 f3f2 b2b3 g5g3 b3c4 f2a2
info multipv 1 depth 4 score cp -541 time 140440342977984 nodes 3946 nps 0 hashfull 1 pv b2b1 g5b5
info multipv 2 depth 4 score cp -647 time 140440342977984 nodes 4078 nps 0 hashfull 1 pv b5b4 f3f2
info multipv 3 depth 4 score cp -687 time 140440342977984 nodes 4400 nps 0 hashfull 1 pv b2c1 f3f1 c1b2 g5g2 b2c3 f1f3 c3c4 g2a2
info multipv 4 depth 4 score cp -687 time 140440342977984 nodes 4537 nps 0 hashfull 1 pv b2a1 f3f1 a1b2 g5g2 b2c3 f1f3 c3c4 g2a2
info multipv 5 depth 4 score cp -743 time 140440342977984 nodes 4716 nps 0 hashfull 1 pv a2a3 g5g2 b2b1 f3a3 a5a4
info multipv 1 depth 5 score cp -675 time 140440342977984 nodes 5758 nps 0 hashfull 1 pv b2c1 f3f1 c1c2 g5g2 c2d3 f1d1 a2d2 g2d2 d3c3
info multipv 2 depth 5 score cp -675 time 140440342977984 nodes 5987 nps 0 hashfull 1 pv b2b1 f3f1 b1c2
info multipv 3 depth 5 score cp -687 time 140440342977984 nodes 5326 nps 0 hashfull 1 pv b2a1 f3f1 a1b2 g5g2 b2c3 f1f3 c3c4 g2a2
info multipv 4 depth 5 score cp -778 time 140440342977984 nodes 6270 nps 0 hashfull 1 pv a2a3 g5g2 b2b1 f3a3 a5a4 h3g4
info multipv 5 depth 5 score cp -778 time 140440342977984 nodes 6413 nps 0 hashfull 1
info multipv 1 depth 6 score mate -53 time 140440342977984 nodes 15683 nps 0 hashfull 5 pv b2c1 g5g1
info multipv 2 depth 6 score mate -22 time 140440342977984 nodes 15857 nps 0 hashfull 5 pv a2a3 g5g2
info multipv 3 depth 6 score mate -3 time 140440342977984 nodes 16032 nps 0 hashfull 5 pv b2a1 g5g1 a1b2 g1g2 b2a1 f3f1
info multipv 4 depth 6 score mate -3 time 140440342977984 nodes 16372 nps 0 hashfull 5 pv b2b1 g5g1 b1b2 g1g2 b2a1 f3f1
info multipv 5 depth 6 score mate -3 time 140440342977984 nodes 16460 nps 0 hashfull 5
info multipv 1 depth 7 score mate -111 time 140440342977984 nodes 24961 nps 0 hashfull 8 pv b2c1 g5g1 c1d2 g1g2 d2e1 f3f1 e1f1 g2g1 f1f2
info multipv 2 depth 7 score mate -4 time 140440342977984 nodes 25109 nps 0 hashfull 8 pv a2a3 g5g2 b2b1 f3a3 a5a4 a3f3 b5b4 f3f1
info multipv 3 depth 7 score mate -3 time 140440342977984 nodes 25199 nps 0 hashfull 8 pv b2b1 g5g1 b1b2 g1g2 b2a1 f3f1
info multipv 4 depth 7 score mate -3 time 140440342977984 nodes 25415 nps 0 hashfull 8 pv b2a1 g5g1 a1b2 g1g2 b2a1 f3f1
info multipv 5 depth 7 score mate -2 time 140440342977984 nodes 25541 nps 0 hashfull 8 pv b5b4 g5g2 b2a1 f3f1
info multipv 1 depth 8 score mate -5 time 140440342977984 nodes 32142 nps 0 hashfull 11 pv b2c1 g5g1 c1d2 g1g2 d2e1 g2a2 a5a4 h3g2 e1d1 f3f1
info multipv 2 depth 8 score mate -5 time 140440342977984 nodes 32232 nps 0 hashfull 11 pv a2a3 g5g2
info multipv 3 depth 8 score mate -4 time 140440342977984 nodes 32292 nps 0 hashfull 11 pv b2a1 g5g1
info multipv 4 depth 8 score mate -3 time 140440342977984 nodes 32842 nps 0 hashfull 11 pv b2b1 g5g1 b1b2 g1g2 b2a1 f3f1
info multipv 5 depth 8 score mate -2 time 140440342977984 nodes 32941 nps 0 hashfull 11 pv b5b4 g5g2 b2a1 f3f1
info multipv 1 depth 8 score mate -5 time 140440342977984 nodes 32142 nps 0 hashfull 11 pv b2c1 g5g1 c1d2 g1g2 d2e1 g2a2 a5a4 h3g2 e1d1 f3f1
info multipv 2 depth 8 score mate -5 time 140440342977984 nodes 32232 nps 0 hashfull 11 pv a2a3 g5g2
info multipv 3 depth 8 score mate -4 time 140440342977984 nodes 32292 nps 0 hashfull 11 pv b2a1 g5g1
info multipv 4 depth 8 score mate -3 time 140440342977984 nodes 32842 nps 0 hashfull 11 pv b2b1 g5g1 b1b2 g1g2 b2a1 f3f1
info multipv 5 depth 8 score mate -2 time 140440342977984 nodes 32941 nps 0 hashfull 11 pv b5b4 g5g2 b2a1 f3f1
bestmove b2c1 ponder g5g2
```

**Comment by jdart1** [comment]
Will look into it. You can expect some instability in the master branch since it just had a big batch of changes related to LazySMP.

**Comment by dsjoerg** [comment]
Thank you.  Is there a branch/tag that you think of as a better bet?  20.5.0?  This particular bug isn't causing me much trouble but I would be happy to use whichever tag/branch you think is the least likely to have trouble lurking.

**Comment by jdart1** [comment]
For right now I would recommend you stick wit the release tag. I expect to have another release out by end of June. That will have LazySMP plus whatever other fixes/changes there are. However I am going to fix this issue and if you need to you can merge that fix into 20.5 in your fork.
